src/backend/crossterm.rs

```diff
@@ -220,88 +223,28 @@ where
 impl CrosstermBackend<Stderr> {
     /// Creates a new `CrosstermBackend` using [`std::io::stderr()`] as the buffer.
     ///
-    /// This method also enables [raw mode] and enters the [alternate screen].
-    ///
     /// # Example
     ///
     /// ```rust,no_run
     /// # use ratatui::backend::{Backend, CrosstermBackend};
-    /// let mut backend = CrosstermBackend::on_stderr()?;
-    /// # std::io::Result::Ok(())
+    /// let mut backend = CrosstermBackend::on_stderr();
     /// ```
-    ///
-    /// [raw mode]: crate::backend#raw-mode
-    /// [alternate screen]: crate::backend#alternate-screen
-    pub fn on_stderr() -> Result<CrosstermBackend<Stderr>, io::Error> {
+    pub fn on_stderr() -> CrosstermBackend<Stderr> {
         CrosstermBackend::new(io::stderr())
-            .with_raw_mode()?
-            .with_alternate_screen()
     }
 }
 
 impl CrosstermBackend<Stdout> {
     /// Creates a new `CrosstermBackend` using [`std::io::stdout()`] as the buffer.
     ///
-    /// This method also enables [raw mode] and enters the [alternate screen].
-    ///
     /// # Example
     ///
     /// ```rust,no_run
     /// # use ratatui::backend::{Backend, CrosstermBackend};
-    /// let mut backend = CrosstermBackend::on_stdout()?;
-    /// # std::io::Result::Ok(())
+    /// let mut backend = CrosstermBackend::on_stdout();
     /// ```
-    ///
-    /// [raw mode]: crate::backend#raw-mode
-    /// [alternate screen]: crate::backend#alternate-screen
-    pub fn on_stdout() -> Result<CrosstermBackend<Stdout>, io::Error> {
+    pub fn on_stdout() -> CrosstermBackend<Stdout> {
         CrosstermBackend::new(io::stdout())
-            .with_raw_mode()?
-            .with_alternate_screen()
-    }
-}
```

Comment:
No longer does raw and alternate for this per feedback - the "easy mode" is moved to the builder.